contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -501,8 +508,11 @@ CAST(Ext.orig_loginname AS sys.nvarchar(128)) AS name,
 CAST('SERVER ROLE' AS sys.nvarchar(128)) AS type,
 CAST ('GRANT OR DENY' as sys.nvarchar(128)) as usage
 FROM pg_catalog.pg_roles AS Base INNER JOIN sys.babelfish_authid_login_ext AS Ext ON Base.rolname = Ext.rolname
-WHERE Ext.type = 'R' AND
-(pg_has_role(sys.suser_id(), 'sysadmin'::TEXT, 'MEMBER'));
+WHERE Ext.type = 'R'
+AND Ext.orig_loginname = 'sysadmin'
+AND bbf_is_member_of_role_nosuper(sys.suser_id(), sys.suser_id('sysadmin'))
+OR Ext.orig_loginname = 'securityadmin'
+AND bbf_is_member_of_role_nosuper(sys.suser_id(), sys.suser_id('securityadmin'));
```

Comment:
login_token should return rows for fixed server role of which the current login is member of.

Reply:
@tanscorpio7 We intentionally added the API which should not consider superuser priv because if superuser is also considers then in BBF for master user if it is only member of sysadmin then also it will bypass the check for securityadmin membership as well and we'll see row for securityadmin as well which is not expected in sql server.I will confirm with dbes.